jit-persistence/src/main/java/org/dcsa/jit/persistence/entity/OperationsEvent.java

```diff
@@ -20,10 +30,57 @@
 @AllArgsConstructor
 @Setter(AccessLevel.PRIVATE)
 @Entity
-@Table(name = "")
+@Table(name = "operations_event")
 public class OperationsEvent {
   @Id
   @GeneratedValue
-  @Column(name = "id", nullable = false)
+  @Column(name = "event_id", nullable = false)
   private UUID id;
+
+  @Column(name = "event_classifier_code", length = 3, nullable = false)
+  private String classifierCode;
+
+  @Column(name = "event_created_date_time", nullable = false)
+  private OffsetDateTime createdDateTime;
+
+  @Column(name = "event_date_time", nullable = false)
+  private OffsetDateTime dateTime;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "publisher", nullable = false)
+  private Party publisher;
+
+  @Enumerated(EnumType.STRING)
+  @Column(name = "publisher_role", nullable = false)
+  private PublisherRole publisherRole;
+
+  @Enumerated(EnumType.STRING)
+  @Column(name = "operations_event_type_code", nullable = false)
+  private OperationsEventTypeCode operationsEventTypeCode;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "event_location", nullable = false)
+  private Location location;
+
+  @ManyToOne(fetch = FetchType.LAZY)
+  @JoinColumn(name = "transport_call_id", nullable = false)
+  private TransportCall transportCall;
```

Comment:
Strictly speaking the API lists as an optional field.  Admittedly, I am not sure what to do in that case. Maybe this is an oversight in swagger.  @HenrikHL do you have a view on that?

Reply:
Is it not null in the database (IM), and that's what the entities are modeling

Reply:
@HenrikHL: tag, you are it on this one.

Reply:
It is an oversight by me - an OperationsEvent would require a link to a TransportCall
This field needs to be @NotNull (I will modify in JIT 1.2)